mcs/class/System.IO.Compression/SharpCompress/IO/ReadOnlySubStream.cs

```diff
@@ -27,6 +28,8 @@ protected override void Dispose(bool disposing)
             }
         }
 
+        private long length { get; set; }
```

Comment:
Right below this line:

```
         private long BytesLeftToRead { get; set; }
```

But yeah, using a private field is more in line with C# conventions.